test/functional/searches_controller_test.rb

```diff
@@ -69,6 +107,38 @@ class SearchesControllerTest < ActionController::TestCase
     should render_template :show
   end
 
+  context 'on GET to show with search parameters and no results' do
+    setup do
+      @sinatra = create(:rubygem, name: "sinatra")
+      @sinatra_redux = create(:rubygem, name: "sinatra-redux")
+      @brando = create(:rubygem, name: "brando")
+      create(:version, rubygem: @sinatra)
+      create(:version, rubygem: @sinatra_redux)
+      create(:version, rubygem: @brando)
+      @sinatra.index_document
+      @sinatra_redux.index_document
+      @brando.index_document
+      Rubygem.__elasticsearch__.refresh_index!
+      get :show, query: "sinatre", es: 'true'
+    end
+
+    should respond_with :success
+    should render_template :show
+    should "see sinatra on the page in the suggestions" do
+      page.assert_text('Maybe you mean')
+      # assert page.find('.search-suggestions').has_content?(@sinatra.name)
```

Comment:
Maybe someone with more capybara experience can show me the right way to do this?